app/frontend/components/pages/accounts/accountDropdown.tsx

```diff
@@ -0,0 +1,46 @@
+import {h} from 'preact'
+import {connect} from '../../../libs/unistore/preact'
+import actions from '../../../actions'
+import {useState, useCallback} from 'preact/hooks'
+
+import range from '../../../wallet/helpers/range'
+import {State} from '../../../state'
+
+const AccountDropdown = ({accountIndex, setAccountFunc, accounts}) => {
+  const [shouldHideAccountDropdown, hideAccountDropdown] = useState(true)
+  const toggleAccountDropdown = useCallback(
+    () => {
+      hideAccountDropdown(!shouldHideAccountDropdown)
+    },
+    [shouldHideAccountDropdown]
+  )
+
+  return (
+    <div className="account-dropdown">
+      <button className="account-dropdown-button" onClick={() => toggleAccountDropdown()}>
+        Account {accountIndex}
+      </button>
+      <div className={`account-dropdown-content ${shouldHideAccountDropdown ? 'hide' : 'show'}`}>
+        {range(0, Object.values(accounts).length).map((i) => (
```

Comment:
looking at wallet.ts it seems to me like accounts have been refactored to a list

Reply:
I think accounts is still an object.